src/patternfly/components/Pagination/examples/Pagination.md

```diff
@@ -127,7 +127,19 @@ import './Pagination.css'
 {{/pagination}}
 ```
 
+### Inset
+```hbs
+{{#> pagination pagination--modifier="pf-m-inset-none pf-m-inset-md-on-md pf-m-inset-2xl-on-lg"}}
```

Comment:
This example isn't currently working since the `2xl` inset isn't supported. If you update `$pf-c-pagination--variable-map` to support all of the spacers as mentioned in another comment, this example will work great!